### PR TITLE
Grab the post from the params since we killed json

### PR DESCRIPTION
**Pull request by rynbyjn**
[Fixes #144130579](https://www.pivotaltracker.com/story/show/144130579)

### CODE REVIEW

File: src/selectors/post.js [hunk 1]
```diff
@@ -25,9 +26,10 @@ export const selectPosts = state => state.json.get(POSTS, Immutable.Map())
 
 // Requires `postId`, `post` or `params.token` to be found in props
 export const selectPost = createSelector(
-  [selectPropsPostId, selectParamsToken, selectPosts], (id, token, posts) => {
+  [selectPropsPost, selectPropsPostId, selectParamsToken, selectPosts],
+  (propsPost, id, token, posts) => {
     if (id) {
-      return posts.get(id, Immutable.Map())
+      return posts.get(id, propsPost || Immutable.Map())
```
Comment (rynbyjn) [inline src/selectors/post.js]: This strays from the webapp a bit, but since we don't have the json state we need to pull the post from the props...

**Comment by rynbyjn** [comment]
@steam yup, it's the story in the description above 🤘

**Comment by steam** [comment]
Sure enough 👀

File: src/selectors/post.js (hunk 1 above)
Comment (mkitt) [inline src/selectors/post.js]: 👍

**Pull request merged**